GiniVisionExample/GiniVisionExample/Screens/Main/MainViewController.swift

```diff
@@ -40,4 +40,8 @@ final class MainViewController: UIViewController {
     @IBAction func showHelp(_ sender: Any) {
         delegate?.main(viewController: self, didTapShowHelp: ())
     }
+    
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        return .portrait
```

Comment:
Have you considered putting that restriction in the info plist?
I know it has some downsides. For instance it doesn't work with the built in image selector, because it crashes.
But on the other hard, it's a less error prone way to ensure nothing is in landscape.

Reply:
That was my first solution, because it is quite simple to specify the keys for both iPad and iPhone.
But I wanted the other non-gvl screens to support also landscape orientation, since there could apps that are using the GVL and they want to support also landscape orientation.

Reply:
Do you think clients would be extending this project? They would probably just import the classes.

But you are right. In this case, it would be nice to have the orientation restricted on OUR view controller so they don't have to worry about it.

Reply:
It is already restricted but if the `GiniNavigationController` is inside of another UINavigationController it won't work, since the parent `UINavigationController` is the one that *decide*.